packages/webapp/components/invite/Referral.tsx

```diff
@@ -0,0 +1,221 @@
+          articles.
+          <span className="tablet:block laptopL:inline"> Stay up to date.</span>
+        </p>
+        <Button
+          buttonSize={isMobile ? ButtonSize.Large : ButtonSize.XLarge}
+          className="p-4 mx-auto laptop:mx-0 mt-6 tablet:mt-12 max-w-[17.5rem] mobileL:max-w-[24.25rem] btn-primary"
+          tag="a"
+          href={downloadBrowserExtension}
+          onClick={() => {
+            handleAcceptClick();
+          }}
+          target="_blank"
+          loading={isLoading}
+          disabled={isLoading || isSuccess}
+        >
+          <FlexCentered className="gap-2 mobileL:typo-title3">
+            <BrowsersIcon
+              width={isMobile ? '58px' : '80px'}
+              height={isMobile ? '30px' : '40px'}
+              className="text-theme-label-primary"
+            />
+            Add to browser - it&apos;s free!
+          </FlexCentered>
+        </Button>
+      </div>
+      <div className="flex fixed flex-auto w-full h-full laptop:initial laptop:bottom-[unset] bottom-[-49%]">
+        <div className="flex relative laptop:absolute justify-center self-center m-auto mx-2 z-10 laptopL:right-[-6rem] laptop:right-[-7.5rem] w-[fit-content] laptop:w-[initial]">
+          <img
+            src={
+              cloudinary.referralCampaign.genericReferral.purpleEdgeGlowTablet
+            }
+            alt="Purple glow right edge"
+            className="fixed laptop:top-0 w-full laptop:w-auto laptop:h-full bottom-[-1rem] laptop:bottom-[unset] laptop:rotate-[270deg] z-20"
+          />
+          <img
+            src={cloudinary.referralCampaign.genericReferral.appScreenshot}
+            alt="Daily.dev app screenshot of my feed page"
+            className="object-contain w-full laptop:w-auto h-auto laptop:h-full z-10 laptopXL:max-h-[39.5rem] laptop:max-h-[25.35rem]"
+          />
+          <img
+            src={cloudinary.referralCampaign.genericReferral.playButton}
+            alt="Play daily.dev introduction video"
+            className="absolute laptop:self-center laptopL:m-auto laptopXL:h-auto laptop:mr-[8.5rem] laptop:mt-[0.5rem] z-20 laptop:h-[8rem] tablet:h-[14.5rem] h-[7.8rem] tablet:mt-[-1.5rem]"
+          />
+          <button
+            className="absolute z-1 w-full h-full"
+            type="button"
+            aria-label="Play daily.dev introduction video"
+            onClick={() => {
+              setVideoOpen(true);
+            }}
+          />
+        </div>
+      </div>
+      {isVideoOpen && (
+        <Modal
+          // eslint-disable-next-line @dailydotdev/daily-dev-eslint-rules/no-custom-color
+          className="px-8 bg-[black]"
+          kind={ModalKind.FlexibleCenter}
+          size={ModalSize.XLarge}
+          isOpen={isVideoOpen}
+          onRequestClose={onVideoModalClose}
+        >
+          <CloseButton
+            buttonSize={ButtonSize.Small}
+            className="top-3 right-3 text-white border-white !absolute !btn-secondary"
```

Comment:
Buttons have a `position` attribute.
Also we force white on all states? (Think yes)

Reply:
Took this piece of code from here https://github.com/dailydotdev/apps/blob/3489d4348c42710c91d50c7257d0351a02366f1f/packages/webapp/pages/join.tsx, probably old so I'll fix this :D

Reply:
> Also we force white on all states? 

Which white are you referring to, text-white?

Reply:
Yeah and the border, it's probably fine, but those are non theme mode supportive (light/dark) just something to keep in mind.

Reply:
So I took this code from the previous video implementation for this page referred from the DR in the video section. If we don't have specific designs on how this button should look like, I believe we can remove those `text-white` and `border-white` as they don't ruin anything, just the X will be in a lighter grey color if not hovered, and there will be no white border.

Reply:
Removed these in the end, just in case we want the border, what would be a white color that would be mode supportive?

Reply:
Technically then it should just be `btn-secondary` not sure we need the overwrite.
Also if both modes have white it's fine to use this.
Otherwise, think primary is the most white one. *(salt.90) i guess?